backend/open_api_spec.yml

```diff
@@ -0,0 +1,762 @@
+          $ref: "#/components/responses/401"
+    post:
+      summary: Create a Collection
+      description: Create a new Collection
+      security:
+        - cxguserCookie: []
+        - curatorAccess: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/collection_form_metadata"
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/uuid"
+        "400":
+          $ref: "#/components/responses/400_multiple"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections/{collection_uuid}:
+    delete:
+      summary: Delete a private Collection
+      description: Delete a private Collection. Delete an existing revision for a public Collection.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      responses:
+        "202":
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                type: string
+        "401":
+          $ref: "#/components/responses/401"
+    get:
+      summary: Fetch a Collection with Datasets
+      description: Fetch Collection metadata and associated Datasets, public and private.
+      security:
+        - cxguserCookie: []
+        - curatorAccess: []
+        - {}
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                  visibility:
+                    $ref: "#/components/schemas/visibility"
+                  contact_name:
+                    type: string
+                  contact_email:
+                    type: string
+                  curator_name:
+                    type: string
+                  revised_at:
+                    type: number
+                  created_at:
+                    type: number
+                  links:
+                    $ref: "#/components/schemas/links"
+                  datasets:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/dataset"
+                  access_type:
+                    type: string
+                    enum: [ READ, WRITE ]
+                  published_at:
+                    type: number
+                  description:
+                    type: string
+                  data_submission_policy_version:
+                    type: string
+                  publisher_metadata:
+                    $ref: "#/components/schemas/publisher_metadata"
```

Comment:
added the publisher_metadata

Reply:
Also, there were columns that were returned from `/curation/collections` which were not being returned here; I've added them to be consistent. Let me know if you disagree @brianraymor @Bento007